calliope/core/model.py

```diff
@@ -125,9 +126,19 @@ def _init_from_model_run(self, model_run, debug_data):
             name='run_config', observer=self._model_data
         )
 
+        # Attach _model_run and _debug_data to _model_data
+        _model_run_to_save = self._model_run.copy()
+        del _model_run_to_save['timeseries_data']  # Can't be serialised!
+        self._model_data.attrs['_model_run'] = _model_run_to_save.to_yaml()
+        self._model_data.attrs['_debug_data'] = self._debug_data.to_yaml()
+
     def _init_from_model_data(self, model_data):
-        self._model_run = None
-        self._debug_data = None
+        self._model_run = AttrDict.from_yaml_string(
```

Comment:
If the above comment were agreed upon, then this would mean deleting it from the attrs after attaching it to the model as `_model_run`. Maybe it isn't worth it then

Reply:
Done